### DISCUSSION TITLE
✅ 🚀 Feature Request: Add ability to resize Accordion Icon

### DISCUSSION THREAD

**Discussion by gregstehle** [elementor/elementor#17585] [Feature Request]
Whilst you are able to add your own accordion icon to expand/contract accordion, there is no feature to change the image size of the icon. I would like to make the icon size much bigger but there is no way to do this. Could this feature please be added in the Appearance settings for the Accordion Icon?

**Comment by nicholaszein** [comment]
Hello there @gregstehle!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new nested Accordion in Elementor v3.15! 🥳 

You can now add custom icons and control their sizes.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
